footprints/settings_shared.py

```diff
@@ -49,7 +50,8 @@
     'footprints.main',
     'geoposition',
     'rest_framework',
-    'reversion'
+    'reversion',
+    'viaf'
```

Comment:
Is viaf going to become a django application? At the moment, there aren't models, views, or other django-specific bits so it would work as just a normal, vanilla python module.